decisioncentral/image.yaml

```diff
@@ -134,10 +134,11 @@ modules:
       version: "7.3-latest"
     - name: jboss.container.eap.setup
     - name: rhdm-7-decisioncentral
-    - name: jboss.container.maven.35.bash
-      version: "3.5"
-    - name: jboss.container.maven.default.bash
-    - name: jboss.container.jolokia.bash
+    - name: jboss.container.maven
+      version: "8.2.3.6"
+    - name: jboss.container.maven.default
+    - name: jboss.container.jolokia
```

Comment:
make sure this modules installs the needed jar and configures it properly like the previous one.

Reply:
Applied the previous one